### DISCUSSION TITLE
Internal - Settings - Always use cache - What does this mean?

### DISCUSSION THREAD

**Discussion by bredmich** [thkl/hap-homematic#463] [General]
Hi there,

i would like to know what the internal setting "always use cache" means.
The Wiki has no information about this setting yet.

kind regards
Michael

**Comment by thkl** [comment]
all values of devices will be cached in hap-homematic .. If HomeKit asks about a device value this cached value will be used.
Every time a device changed a value the ccu emits an event and hap will update that cache. disabling this will end up in a request to the ccu for this device datapoint value every time HomeKit asks for a value.

enabling this will reduce the load on your ccu but may end up in outdated datapoint values when any of the events gets lost.
